ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -156,9 +169,11 @@ impl<'a> TraceBuilder<'a> {
 
         // Decide how to translate each AOT instruction based upon its opcode.
         for (inst_idx, inst) in blk.instrs.iter().enumerate() {
+            println!("inst: {}", inst.to_string(self.aot_mod));
```

Comment:
left-in debug?

Reply:
[72bb808](https://github.com/ykjit/yk/pull/1082/commits/72bb808055c45ac6daf6e62202ceafbf2dd68c2e)